decidim-dev/lib/decidim/test/rspec_support/capybara.rb

```diff
@@ -42,7 +42,7 @@ def switch_to_default_host
 }
 
 Capybara.register_driver :poltergeist do |app|
-  Capybara::Poltergeist::Driver.new(app, capybara_options)
+  Capybara::Poltergeist::Driver.new(app, capybara_options.merge(screen_size: [1600, 1200]))
```

Comment:
This is no actual valid resolution. [As of January 2016](http://www.w3schools.com/browsers/browsers_display.asp) it seems to be 1366x768.